common/source/docs/common-wiki-editing-setup.rst

```diff
@@ -41,8 +41,8 @@ Be sure to keep your fork of the repository updated, both locally, and on GitHub
        git rebase upstream/master
        git push -f origin master
        
-Setup in Linux or Windows WSL
------------------------------
+Setup in Linux
+-------------------------
```

Comment:
one little nit and will merge....underlines must be same length as text above...thanks

Reply:
Fixed!